src/main/java/ibas/inchelin/domain/notice/service/NoticeService.java

```diff
@@ -0,0 +1,37 @@
+package ibas.inchelin.domain.notice.service;
+
+import ibas.inchelin.domain.notice.entity.Notice;
+import ibas.inchelin.domain.notice.repository.NoticeRepository;
+import ibas.inchelin.web.dto.notice.NoticeContentResponse;
+import ibas.inchelin.web.dto.notice.NoticeListResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class NoticeService {
+
+    private final NoticeRepository noticeRepository;
+
+    // 공지 목록 조회
+    public List<NoticeListResponse.NoticeListDto> findNotices() {
```

Comment:
기존 DTO 구조에 맞게 리팩터링이 필요할 것 같습니다.
```suggestion
    public NoticeListResponse findNotices() {
```